src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining.UnitTests/Application/Courses/Queries/GetCourseByLarsCode/WhenGettingCourseByLarsCode.cs

```diff
@@ -495,4 +438,154 @@ GetCourseTrainingProvidersCountResponse courseTrainingProvidersCountResponse
             x.GetCachedLocationInformation(query.Location, false
         ), Times.Once);
     }
+
+
+    [TestCase(-1, 1, 1, null, 6)]
+    [TestCase(-1, 1, 1, 2, 6)]
+    [TestCase(1, null, -1, 1, 15)]
+    [TestCase(1, 2, -1, 1, 15)]
+    [TestCase(1, 2, 2, 5, 0)]
+    [TestCase(-10, -5, -20, -15, 0)]
+    public async Task Handle_Returns_Correct_IncentivePayment(int firstStartDateIncrement, int? firstEndDateIncrement,
```

Comment:
TestCase is passing in more parameters than the method demands? Can this test be more descriptive, it seems cryptic to follow.

Reply:
No it's 5 parameters, as expected.  I'll try and rework to make it clear it's dealing with past and future dates